curve-ansible/roles/install_package/templates/nebd-daemon.j2

```diff
@@ -10,261 +10,305 @@
 # Description:       Start the nebd-daemon service and associated helpers
 ### END INIT INFO
 
-# 检查脚本执行是否通过root权限执行
-if [[ $(id -u) -ne 0 ]]
-then
-    echo "Please run with sudo"
-    exit 1
-fi
-
-# nebd-server路径
-bin={{ curve_bin_dir }}/nebd-server
-
-# 默认配置文件
-confPath={{ nebd_server_config_path }}
-
-# 日志文件路径
-baseLogPath={{ nebd_log_dir }}
-logPath=${baseLogPath}/server
-
-# pidfile
-pidFile=${baseLogPath}/nebd-server.pid
+NEBD_SERVER_BIN=/usr/bin/nebd-server
```

Comment:
Before the nebd-daemon.j2 will use client.ini to generate the nebd-daemon.sh. If it has difficult to generate from jinja template, can the field like 'nebd_data_dir'  be delete ?

Reply:
`curve_bin_dir` was added backup, and another fileds now have default value.